maplibre/src/lib.rs

```diff
@@ -73,6 +93,12 @@ where
     SM: ScheduleMethod,
     HC: HTTPClient,
 {
+    /// Initializes the whole rendering pipeline for the given configuration.
+    /// Returns the initialized map, ready to be run.
+    ///
+    /// # Panics
+    ///
+    /// * Panics if Winit is unable to retrieve the target window size.
```

Comment:
Which panic do you refer here to?

Reply:
I think after this task all panics should vanish: https://github.com/maplibre/maplibre-rs/issues/92